### PR TITLE
chore(owners): add new reviewers

### PR DESCRIPTION
**Pull request by kim-tsao**
## Description

Please explain the changes you made here.

## Which issue(s) does this PR fix

- Fixes #?
- https://issues.redhat.com/browse/RHIDP-3124

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -14,3 +14,8 @@ approvers:
  - Zaperex
 
 reviewers:
+ - 04kash
+ - dzemanov 
+ - Fortune-Ndlovu
+ - JessicaJHee
```
Comment (rm3l) [inline OWNERS:20]:
```suggestion
 - JessicaJHee
 - rm3l
```

May I be added as a reviewer too?

Comment (its-mitesh-kumar) [reply OWNERS:20]: Please add me as reviewer as well : its-mitesh-kumar

File: OWNERS [hunk 2]
```diff
@@ -8,9 +8,16 @@ approvers:
  - kim-tsao
  - nickboldt
  - PatAKnight
+ - r3ml
```
Comment (rm3l) [inline OWNERS:11]:
```suggestion
 - rm3l
```

Typo with my username.

**Pull request merged**